docs/source/howitworks.rst

```diff
@@ -24,10 +24,15 @@ object is instantiated:
    )
 
 You then ask for more workers using the ``scale`` command:
+You can either specify the number of jobs, or the total number of cores or
```

Comment:
While you are touching this file, I think it would improve the doc significantly if there was a precision about two things:
* `FooCluster` arguments are per-job characteristics, i.e. 24 cores per job, `'100GB'` per job
* creating a `FooCluster` object does not launch any jobs, you need to use `.scale` for launching jobs

That can be done in a separate PR though.